authentik/sources/plex/api/source.py

```diff
@@ -109,7 +109,11 @@ def redeem_token(self, request: Request) -> Response:
                 source=source,
                 request=request,
                 identifier=str(identifier),
-                enroll_info=user_info,
+                enroll_info={
+                    "user_info": user_info,
+                },
+                groups_info=[],
```

Comment:
Same as with discord, we could fetch a list of all the users' servers or something and provide that as group info